figs/paper/paper.Rmd

```diff
@@ -0,0 +1,101 @@
+- name: Mine Çetinkaya-Rundel
+  orcid: 0000-0001-6452-2420
+  affiliation: "2, 7"
+affiliations:
+ - name: Johns Hopkins, Department of Biostatistics
+   index: 1
+ - name: RStudio
+   index: 2
+ - name: UC Berkeley, Department of Statistics and Reed College Mathematics Department (on leave)
+   index: 3
+ - name: Flatiron School
+   index: 4
+ - name: No Affiliation
+   index: 5
+ - name: Smith College, Program in Statistical & Data Sciences
+   index: 6
+ - name: Duke University, Department of Statistical Science
+   index: 7
+
+citation_author: Couch et. al.
+date: 12 June 2021
+year: 2021
+bibliography: paper.bib
+output: 
+  rticles::joss_article:
+    keep_tex: true
+    includes:
+      in_header: columns.tex
+csl: apa.csl
+journal: JOSS
+---
+
+# Summary
+
+`infer` implements an expressive grammar to perform statistical inference that adheres to the `tidyverse` design framework [@wickham2019welcome]. Rather than providing methods for specific statistical tests, this package consolidates the principles that are shared among common hypothesis tests into a set of four main verbs (functions), supplemented with many utilities to visualize and extract value from their outputs.
+
+# Statement of Need
+
+Packages implementing methods for basic statistical inference in R are highly variable in their interfaces. The structure of inputted data, argument names, expected argument types, argument orders, output types, and spelling cases varies widely both within and among packages. This diversity in approaches obscures the intuition shared among common inferential procedures, makes details of usage difficult to remember, and prevents an expressive and idiomatic coding style.
+
+`infer` is an R package for randomization-based hypothesis testing, naturalizing an intuitive understanding of statistical inference via a unified and expressive grammar. Four functions provide functionality encompassing a large swath of basic frequentist statistical inference, abstracting away details of specific tests and shifting the focus of the analyst to the observed data and the processes that generated it. Such a grammar lends itself to applications in teaching, data pedagogy research, applied scientific research, and advanced predictive modeling. For one, the principled approach of the package has made it an especially good fit for teaching introductory statistics [@ismay2019statistical; @baumer2020teaching] and research in data pedagogy [@cetinkaya2021fresh; @fergusson2021introducing; @loy2021bringing]. Further, the package has already seen usage in a number of published scientific applications [@mclean2021controlled; @ask2021per; @fallon2021single]. Finally, the package integrates with the greater tidymodels collection of packages, a burgeoning software ecosystem for tidyverse-aligned predictive modeling used across many modern research and industrial applications [@kuhn2020tidymodels]. To date, the package has been downloaded more than 400,000 times.
```

Comment:
@cetinkaya2021fresh is about a data science course so I wonder if it might make more sense to phrase as 

> the principled approach of the package has made it an especially good fit for teaching introductory statistics and data science [@ismay2019statistical; @baumer2020teaching, @cetinkaya2021fresh] and ...